helm-chart/values.yaml

```diff
@@ -0,0 +1,135 @@
+# Number of replicas (instances) to deploy
+replicaCount: 1
+
+# Kubernetes namespace where resources will be deployed
+namespace: gatling
+
+controlPlane:
+  # Name of the control plane deployment
+  name: gatling-cp
+  # Authentication token for the control plane
+  token: token
+  # Description of the control plane
+  description: My K8s control plane description
+  image:
+    # Docker image name for the control plane
+    name: gatlingcorp/control-plane:latest
+    # Image pull policy (Always, IfNotPresent, Never)
+    pullPolicy: IfNotPresent
+    # Image pull secrets for private registry (list of secrets)
+    pullSecrets: []
+  # Additional labels to add to the deployment
+  deploymentLabels: {}
+  # Additional labels to add to the pods
+  podLabels: {}
+  # List of init containers to run before the app containers
+  initContainers: []
+  # Environment variables to set in the container
+  env: []
+  # Command to override default entrypoint of the container
+  command: []
+  # Volumes to attach to the pod
+  volumes: []
+  # Volume mounts for the container
+  volumeMounts: []
+  # Node selector to constrain pods to nodes with specific labels
+  nodeSelector: {}
+  # Affinity rules for pod scheduling
+  affinity: {}
+  # Tolerations to allow pods to be scheduled on tainted nodes
+  tolerations: {}
+  resources: 
+    # Resource requests for the container
+    requests:
+      memory: "512Mi"
+      cpu: "1"
+    # Resource limits for the container
+    limits:
+      memory: "512Mi"
+      cpu: "1"
+  # Security context for the pod or container
+  securityContext: {}
+  #extra_content: {}  # Extra content (currently commented out)
+
+privateLocations:
+  - id: "prl_kubernetes"  # Unique identifier for the private location
+    description: "Private Location on Kubernetes"  # Description of the private location
+    type: "kubernetes"  # Type of the private location
+    engine: "classic" # Execution engine type (classic or javascript)
+    image:
+      type: "certified" # Image type (certified or custom)
+      #java: "latest"  # Java version to use (optional if engine = java, noop if engine type = javascript)
+      #image: "gatlingcorp/classic-openjdk:latest"  # Custom image name
+    #systemProperties: {}  # System properties for the JVM
+    #javaHome: "/usr/lib/jvm/zulu"  # Java home path
+    #jvmOptions: ["-Xmx4G", "-Xms512M"]  # JVM options 
+    #keepLoadGeneratorAlive: false  # Whether to keep the load generator alive (for debugging)
+    job:
+      spec:
+        template:
+          metadata:
+            annotations: {}  # Annotations to add to the pod
+            labels: 
+              # Labels to add to the pod
+              helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+            namespace: "{{ .Values.namespace }}"  # Namespace for the pod
+          spec:
```

Comment:
I believe we need to add more comment regarding job spec. 

We should include a link to [this example JSON job definition](https://docs.gatling.io/reference/install/cloud/private-locations/kubernetes/configuration/#example-json-job-definition) and highlight that we’re working at the template spec level here, rather than the job spec level.